clients/web/src/templates/body/userList.pug

```diff
@@ -7,6 +7,9 @@
     button.g-user-create-button.btn.btn-sm.btn-default
       i.icon-plus-squared
       |  Create user
+    i.icon-user
+      span.badge
```

Comment:
@jeffbaumes noticed that this is in italics -- that's because the text is inside the `i` tag, it should be a sibling instead.